TLM/TLM/CodeProfiler.cs

```diff
@@ -1,84 +1,96 @@
-﻿using ColossalFramework;
-using System;
-using System.Collections.Generic;
-using System.Diagnostics;
-using System.Linq;
-using System.Text;
-using System.Threading;
-
-namespace TrafficManager {
+﻿namespace TrafficManager {
 #if TRACE
-	public class CodeProfiler : Singleton<CodeProfiler> {
-		private Dictionary<string, Stopwatch> watches = new Dictionary<string, Stopwatch>();
-		private Dictionary<string, ulong> intervals = new Dictionary<string, ulong>();
+    using System.Collections.Generic;
+    using System.Diagnostics;
+    using System.Threading;
+    using ColossalFramework;
+    using CSUtil.Commons;
+
+    public class CodeProfiler
+        : Singleton<CodeProfiler>
+    {
+        private readonly Dictionary<string, Stopwatch> watches_ = new Dictionary<string, Stopwatch>();
+        private readonly Dictionary<string, ulong> intervals_ = new Dictionary<string, ulong>();
 
-		internal void Start(string name) {
-			Validate(name);
-			try {
-				Monitor.Enter(watches);
-				watches[name].Start();
-			} finally {
-				Monitor.Exit(watches);
-			}
-		}
+        internal void Start(string name1) {
```

Comment:
`name1` ?

Reply:
Parameter name conflicts with existing parent class field, i had no imagination to rename it.